src/program.js

```diff
@@ -92,15 +101,8 @@ export class Program {
     // This is a convenience for setting global options.
     // An option is only global (i.e. available to all sub commands)
     // with the `global` flag so this makes sure every option has it.
-    this.defaultValues = {};
+    this.defaultValues = setDefaultValues(options);
```

Comment:
Since `this.defaultValues` is mutated in several places, you'll need to be a little bit more careful about setting it.

First, initialize it in the `constructor()` like this:
````js
this.defaultValues = {};
````

Next, make sure you are preserving the old values on this line:
````js
this.defaultValues = {...this.defaultValues, setDefaultValues(options)};
````